### PR TITLE
Add Review Reminder

### PR DESCRIPTION
**Pull request by toviet**
Reviews are in here https://github.com/mention-me/pivotal-lint/pull/1

Screenshot 2020-06-24 at 21 55 50

### CODE REVIEW

**Comment by toviet** [comment]
@edhgoose I decided to create a new branch with better name. Unfortunately the review comments will only be in the old PR

File: src/utils.ts [hunk 1]
```diff
@@ -252,6 +279,14 @@ const getStoryIcon = (storyType: string): string => {
   }
 };
 
+const getPivotalReviewReminders = (reviews: Review[], memberships: PivotalProjectMembership[]): string[] => {
+  return reviews.map((review) => {
+    const member = memberships.find(member => member.person.id && member.person.id === review.reviewer_id);
+
+    return `- [ ] ${review.review_type.name} ${member && member.person && member.person.name ? `(${member.person.name})`: ''}`;
```
Comment (edhgoose) [inline src/utils.ts:286]: Would it be worth doing this?

```suggestion
    return `- [ ] ${review.review_type.name} ${member && member.person && member.person.name ? `(${member.person.name})`: 'Unknown'}`;
```

What's the case where the name isn't known? Does that happen if no reviewer is set? Or are there other cases?

**Review by edhgoose — APPROVED** [review]
Nice! Looks good @toviet. One small suggestion I'd recommend, but otherwise, I'd say good to go. 

Have a look at the comment, and when you're happy, hit merge!

File: src/utils.ts (hunk 1 above)
Comment (toviet) [reply src/utils.ts:286]: if there isn't any person assigned to the review, it would just be [ ] Designer. The code should already cover that

**Pull request merged**